src/types/domains.ts

```diff
@@ -39,19 +39,19 @@ export const hexMap: Record<Color, string> = {
 };
 
 export const graphQlDomainColorMap: Record<string, Color> = {
-  [DomainColor.LIGHTPINK]: Color.LightPink,
-  [DomainColor.PINK]: Color.Pink,
-  [DomainColor.BLACK]: Color.Black,
-  [DomainColor.EMERALDGREEN]: Color.EmeraldGreen,
-  [DomainColor.BLUE]: Color.Blue,
-  [DomainColor.YELLOW]: Color.Yellow,
-  [DomainColor.RED]: Color.Red,
-  [DomainColor.GREEN]: Color.Green,
-  [DomainColor.GOLD]: Color.Gold,
-  [DomainColor.AQUA]: Color.Aqua,
-  [DomainColor.BLUEGREY]: Color.BlueGrey,
-  [DomainColor.PURPLE]: Color.Purple,
-  [DomainColor.ORANGE]: Color.Orange,
-  [DomainColor.MAGENTA]: Color.Magenta,
-  [DomainColor.PURPLEGREY]: Color.PurpleGrey,
+  [DomainColor.Lightpink]: Color.LightPink,
+  [DomainColor.Pink]: Color.Pink,
+  [DomainColor.Black]: Color.Black,
+  [DomainColor.Emeraldgreen]: Color.EmeraldGreen,
+  [DomainColor.Blue]: Color.Blue,
+  [DomainColor.Yellow]: Color.Yellow,
+  [DomainColor.Red]: Color.Red,
+  [DomainColor.Green]: Color.Green,
+  [DomainColor.Gold]: Color.Gold,
+  [DomainColor.Aqua]: Color.Aqua,
+  [DomainColor.Bluegrey]: Color.BlueGrey,
+  [DomainColor.Purple]: Color.Purple,
+  [DomainColor.Orange]: Color.Orange,
+  [DomainColor.Magenta]: Color.Magenta,
+  [DomainColor.Purplegrey]: Color.PurpleGrey,
 };
```

Comment:
>Maybe an IIFE that generates the object at run time ?

But we'd still need the mapping somewhere, right?

The only improvement I can think of is to start using the GraphQL's `DomainColor` enum instead of a custom one, that way we'll only need to maintain the mapping between `DomainColor` and hex colors, instead of DomainColor <> Color <> hex mappings